manifest.php

```diff
@@ -33,11 +33,11 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '20.0.1',
+    'version'     => '20.1.0',
     'author'      => 'Open Assessment Technologies',
     'requires' => array(
         'taoItems' => '>=6.6.0',
-        'tao'      => '>=34.0.0',
+        'tao'      => '>=34.3.0',
```

Comment:
Please update to version `>=35.0.0` https://github.com/oat-sa/tao-core/pull/2108/files#r285632537

Reply:
Please consider also changing the branch name to `feature/NEX-89_modify-references-tao-core`